README.md

```diff
@@ -542,33 +522,6 @@ const const name = "Luke"¡
 print(name)! // "Lu"
 ```
 
-## Semantic naming
-
-DreamBerd supports semantic naming.
-
-```java
-const const sName = "Lu"!
-const const iAge = 29!
-const const bHappy = true!
-```
-
-**New for 2023:** You can now make globals!
-
-```java
-const const g_fScore = 4.5!
-```
-
-## Reversing
```

Comment:
Don't remove reversing as it wouldn't make loop because the first reverse would happen before the second and therefor the second wouldn't be reached